package.json

```diff
@@ -14,6 +14,7 @@
     "nodemon": "^2.0.22"
   },
   "dependencies": {
+    "express-graphql": "^0.12.0",
```

Comment:
do we need na `express-graphql`, can't we just use the plain `graphql` package?

Reply:
no sir, delete ko na po.